index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="./style.css" />
+    <title>Personal Portfolio</title>
+  </head>
+  <body>
+    <header>
+      <nav>
+        <a href="#" class="logo">My logo</a>
+        <img src="./img/menu.svg" alt="" />
+      </nav>
+    </header>
+
+    <section class="content">
+      <div class="subcontent">
+        <h1>
+          I'm Idrissa Omar <br />

```

Comment:
[OPTIONAL]

- [ ] Please take into consideration keeping only one name in your title as it makes your design not look very well on a small device; Also, I recommend this in order to respect the Figma design

Your design:
![image](https://user-images.githubusercontent.com/83596135/165175490-91a78e2a-2734-4aee-a1a6-b1da486623e8.png)